src/ResourceManager/Consumption/Commands.Consumption/Models/PSUsageDetail.cs

```diff
@@ -22,25 +21,32 @@ namespace Microsoft.Azure.Commands.Consumption.Models
 {
     public class PSUsageDetail
     {
+        public string AccountName { get; set; }
+        public string AdditionalProperties { get; set; }
+        public decimal? BillableQuantity { get; set; }
+        public string BillingPeriodId { get; set; }
+        public string ConsumedService { get; set; }
+        public string CostCenter { get; set; }
+        public string Currency { get; set; }
+        public string DepartmentName { get; set; }
         public string Id { get; private set; }
-        public string Name { get; private set; }
```

Comment:
Renaming properties is a breaking change - please add back InvoiceName and BillingPeriodName

Reply:
Sorry for confusion of shuffling the parameters on alphabetic order. InvoiceName and BillingPeriodName were added back.